### ISSUE TITLE
Why is this deprecated now?

### ISSUE THREAD

**Issue by CodeBradley** [OwlCarousel2/OwlCarousel2#2672]
I've been using OC for a while now and I'm just now noticing the deprecation notice and the Tiny slider notice. How long ago did this change and why?

**Comment by cbrophy78** [comment]
Same here
@CodeBradley looks like it was depreciated on [Jul 31](https://github.com/OwlCarousel2/OwlCarousel2/commit/4eedccac4ea061931162a86e3f268332c16a1ad0)

**Comment by JayHoltslander** [comment]
I'm curious to know what happened/why as well.
Tiny-Slider isn't working as well for me in my tests.

**Comment by cbrophy78** [comment]
Tiny-Slider was not working in IE11 for me. Took a week to figure that out. I moved to glide.js but there is a lot to digest there to get what i had with this.

**Comment by dkoether** [comment]
Same here, I tried to find the best alternative for owl carousel with the same browser support but I didn't found a good one. Swiper.js is the closest. It seems like that there is a mass extinction of sliders currently.

**Comment by CodeBradley** [comment]
Yeah, I've still been using it in projects because I don't like the alternatives. Ill have to check out Swiper.js.. RIP OwlCarousel. If anyone knows of the best alt repo then please let us know 🧐

**Comment by peterpp** [comment]
I've just migrated my project from OwlCarousel to Keen-Slider: https://github.com/rcbyr/keen-slider
It is very light-weight, universal and works well. It lacks integrated navigation and auto-play, but it can be added with some effort. Many use-cases are demonstrated in demos.
